lib/src/async_callback.c

```diff
@@ -0,0 +1,136 @@
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <dart_api.h>
+#include <dart_api_dl.h>
+
+#include "libwormhole_william.h"
+// TODO define this in the build
+#define DEBUG
```

Comment:
Move to CMakeLists

Reply:
Implementation added to the bindings CMakeLists.txt file.